CondFormats/GEMObjects/src/GEMeMap.cc

```diff
@@ -81,13 +81,16 @@ void GEMeMap::convertDummy(GEMROMapping& romap) {
   for (int re = -1; re <= 1; re = re + 2) {
     for (int st = GEMDetId::minStationId0; st <= GEMDetId::maxStationId; ++st) {
```

Comment:
If you use a `unsigned` you probably don't need a `static_cast`. Station numbers are positive after all.

Reply:
enum classes are strongly typing, so even with an unsigned, a cast is necessary (just double checked, and it won't compile with an unsigned int or uint16_t either)

Reply:
In that case, use `GEMSubDetId::station(st) == GEMSubDetId::Station::GE21` or `GEMSubDetId::station(st) == GEMSubDetId::Station::GE0` in combination with `unsigned st`

Reply:
Updated, but I think its better to keep it as an int, since its being used to generate the GEMDetId, which takes in an int